R/as-tsibble.R

```diff
@@ -283,7 +283,8 @@ build_tsibble_meta <- function(
 ) {
   stopifnot(is_string(index) && is_string(index2))
   stopifnot(!is_null(ordered))
-  tbl <- as_tibble(x)
+  stopifnot(tibble::is_tibble(x))
```

Comment:
no need for this line.

Reply:
This isn't expensive. If the check is dropped, then shouldn't the other `stopifnot` checks be dropped?

Reply:
Fine. But can we only test if it's a data frame?